src/bootstrap-maxlength.js

```diff
@@ -50,27 +50,32 @@
       }
       options = $.extend(defaults, options);
 
+
+      /**
+      * Return the byte count of the specified character in UTF8 encoding.
+      * Note: This won't cover UTF-8 characters that are 4 bytes long.
+      *
+      * @param input
+      * @return {number}
+      */
+      function utf8CharByteCount(character) {
+        var c = character.charCodeAt();
+        // Not c then 0, else c < 128 then 1, else c < 2048 then 2, else 3
+        return !c ? 0 : c < 128 ? 1 : c < 2048 ? 2 : 3;
```

Comment:
If you have to write a comment to explain what you're doing it is probably more readable an if even if less elegant.